go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.2.0
 	k8s.io/api v0.23.4
 	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v1.5.2
+	k8s.io/client-go v0.23.3
```

Comment:
The other deps are v0.23.4 -- should this match?

Reply:
oh yes my bad.. I can't type..